File: API/DBAccess/Entities/CartLine.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace API.DBAccess.Entities
{
    public class CartLine
    {
        public int Id { get; set; }
        [Required]
        public double Quantity { get; set; }
        [Required]
        public int ProductId { get; set; }
        public Product Product { get; set; }
        [Required]
        public Guid CartId { get; set; }
        public Cart Cart { get; set; }
        [Required]
        public DateTime CreateDate { get; set; }
        [Required]
        public DateTime ModDate { get; set; }
    }
}

File: client/src/app/_models/cartLine.ts
import { ActualPrice } from "./actualPrice";

export class CartLine {
    id:number=0
    productId:number=0
    cartId:string='';
    quantity:number=0;
    photoUrl:string='';
    name:string='';
    actualPrice:ActualPrice;
    price:number;
   
    constructor(){}
}